telegram/forcereply.py

```diff
@@ -44,8 +44,8 @@ class ForceReply(ReplyMarkup):
         **kwargs (:obj:`dict`): Arbitrary keyword arguments.
 
     Attributes:
-        force_reply (:obj:`True`): Shows reply interface to the user, as if they manually selected
-            the bots message and tapped 'Reply'.
+        force_reply (:obj:`bool`): Shows reply interface to the user, as if they manually selected
+            the bots message and tapped 'Reply'. Defaults to :obj:`True`.
```

Comment:
are you sure thats the only occurrence of this in the docs, and are you sure you want to change it? I liked it the way it was :(

Reply:
sameee, it wasn't technically correct, but it looked nice actually, I'll revert